.github/workflows/heroku_deploy.yml

```diff
@@ -0,0 +1,50 @@
+# This is a basic workflow to help you get started with Actions
+name: Dev deploy
+# Controls when the workflow will run
+on:
+  # Runs on the push to the develop branch
+  push:
+    branches:
+      - develop
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
There is nothing in the workflow_dispatch. I think it's better to remove this

Reply:
Actually not. The reason for that is for a user to call that action on demand. Without that empty entry it will only be triggered by the predefined actions like push